### ISSUE TITLE
toInetAddress() question

### ISSUE THREAD

**Issue by tapir** [seancfoley/IPAddress#15]
I'm trying this
```java
import inet.ipaddr.*;

public class MyClass {
    public static void main(String args[]) {
        try {
            IPAddress addr = new IPAddressString("::/0").toAddress();
            System.out.println(addr.getNetworkPrefixLength());
            System.out.println(addr.toInetAddress().getHostAddress());
        } catch(Exception e) {
            System.out.println(e);
        }
    }
}

```
which gives
```
0
ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff
```
But according to documentation `toInetAddress()` should give the lowest of the range which I suppose should be `0:0:0:0:0:0:0:0` no?

I have no idea about CIDR and IP in general so If I'm mistaken apologies in advance.

**Comment by seancfoley** [comment]
You're right, that's a bug with toInetAddress().  Will fix very shortly.  As a workaround, you can do:

    addr.toLower().toInetAddress().getHostAddress()

**Comment by seancfoley** [comment]
This is fixed in release 5.0.2, which is in maven central repo
